tests/bsim/bluetooth/ll/_compile_permutate_kconfigs.sh

```diff
@@ -14,8 +14,9 @@ DEBUG_PERMUTATE=false
  directory}"
 
 source ${ZEPHYR_BASE}/tests/bsim/compile.source
-
+BOARD="${BOARD:-nrf52_bsim/native}"
```

Comment:
These 2 lines where just removed in another commit.
Seems these 2 lines were just added back in a bad merge?